Packs/Sepio/Integrations/SepioPrimeAPI/SepioPrimeAPI.py

```diff
@@ -0,0 +1,1216 @@
+        return {'Authorization': f'Bearer {token}'}
+
+    @staticmethod
+    def __prime_handle_http_response(http_res, url_suffix):
+        if http_res.status_code == 400:
+            raise Exception(http_res.text)
+
+        if http_res.status_code == 403:  # forbbiden for users with this type of user
+            raise Exception('This command can be used only by Sepio Prime '
+                            'users with higher user profile')
+
+        if not http_res.ok:
+            raise Exception(
+                f'Failed to request {url_suffix}, reason: ({http_res.status_code}) {http_res.reason}: {http_res.text}')
+
+
+def parse_speio_prime_event_raw_info(event):
+    desc = event['description']
+    parsers = None
+    if desc == 'New USB peripheral detected':
+        parsers = EVENT_CONTENT_PARSER['newUSBPeripheralDetected']
+
+    if parsers:
+        for event_field, parser in parsers.items():
+            parser_match = parser.search(event[event_field])
+            if parser_match:
+                event.update(parser_match.groupdict())
+
+
+def convert_to_demisto_severity(severity: str) -> int:
+    """Maps Sepio Prime Events severity to Cortex XSOAR severity
+
+    Converts the SepioPrimeAPI alert severity level ('Debug', 'Notice',
+    'Informational', 'Alert', 'Warning', 'Error', 'Emergency', 'Critical') to Cortex XSOAR incident severity (1 to 4)
+    for mapping.
+
+    :type severity: ``str``
+    :param severity: severity as returned from the Sepio Prime event (str)
+
+    :return: Cortex XSOAR Severity (1 to 4)
+    :rtype: ``int``
+    """
+
+    return SEPIO_PRIME_SEVERITY_TO_DEMOISTO_SEVERITY_CONVERT[severity]
+
+
+def arg_to_int(arg, arg_name, required):
+    """Converts an XSOAR argument to a Python int
+
+    This function is used to quickly validate an argument provided to XSOAR
+    via ``demisto.args()`` into an ``int`` type. It will throw a ValueError
+    if the input is invalid. If the input is None, it will throw a ValueError
+    if required is ``True``, or ``None`` if required is ``False.
+
+    :type arg: ``Any``
+    :param arg: argument to convert
+
+    :type arg_name: ``str``
+    :param arg_name: argument name
+
+    :type required: ``bool``
+    :param required:
+        throws exception if ``True`` and argument provided is None
+
+    :return:
+        returns an ``int`` if arg can be converted
+        returns ``None`` if arg is ``None`` and required is set to ``False``
+        otherwise throws an Exception
+    :rtype: ``Optional[int]``
+    """
+
+    if arg is None:
+        if required is True:
+            raise ValueError(f'Missing "{arg_name}"')
+        return None
+    if isinstance(arg, str):
+        if arg.isdigit():
+            return int(arg)
+        raise ValueError(f'Invalid number: "{arg_name}"="{arg}"')
+    if isinstance(arg, int):
+        return arg
+    raise ValueError(f'Invalid number: "{arg_name}"')
+
+
+def arg_to_timestamp(arg, arg_name, required):
```

Comment:
Youre right, i think it may be useful although it not used yet.

I think to keep it for future updates but its not a must, @Itay4 what do you think ?

Reply:
you can keep it if you'd like

Reply:
Ok